feathr_project/feathr/api/app/backends/backend_base.py

```diff
@@ -0,0 +1,122 @@
+from abc import abstractmethod,ABC
+
+
+class FeathrBackendAbstract(ABC):
```

Comment:
add documentation for each method. if not needed for now, just remove them for now and u can always add it back in future PRs.

Reply:
Changed